backends/tfhe-cuda-backend/cuda/include/integer/integer_utilities.h

```diff
@@ -4981,6 +4981,82 @@ template <typename Torus> struct int_sub_and_propagate {
   }
 };
 
+template <typename Torus> struct int_extend_radix_with_sign_msb {
+
+  int_radix_params params;
+  bool allocate_gpu_memory;
+
+  int_radix_lut<Torus> *lut;
+
+  CudaRadixCiphertextFFI *last_block;
+  CudaRadixCiphertextFFI *padding_block;
+
+  int_extend_radix_with_sign_msb(
+      cudaStream_t const *streams, uint32_t const *gpu_indexes,
+      uint32_t gpu_count, const int_radix_params params, uint32_t num_luts,
```

Comment:
Why do you have a num_luts argument here? I wouldn't expect it to be necessary, is it?